Cargo.toml

```diff
@@ -25,51 +21,26 @@ travis = []
 with-syntex = ["syntex", "pnet_macros/with-syntex"]
 
 [dependencies]
+libc = "0.2"
 ipnetwork = "0.12"
+
 pnet_base = { path = "pnet_base" }
 pnet_sys = { path = "pnet_sys" }
 pnet_datalink = { path = "pnet_datalink" }
+pnet_macros = { path = "pnet_macros", optional = true, version = ">=0.9" }
+pnet_macros_support = { path = "pnet_macros_support", version = ">=0.1" }
+pnet_macros_plugin = { path = "pnet_macros_plugin", optional = true, version = ">=0.1" }
 
-[dependencies.clippy]
-optional = true
-version = ">=0.0"
-
-[dependencies.netmap_sys]
-features = ["netmap_with_libs"]
-optional = true
-version = ">=0.0"
-
-[dependencies.libc]
-version = "0.2.*"
-
-[dependencies.syntex]
-version = "0.42.*"
-optional = true
-
-[dependencies.pcap]
-version = "0.6.*"
-optional = true
-
-[dependencies.pnet_macros]
-optional = true
-default-features = false
-features = []
-path = "pnet_macros"
-version = ">=0.9"
-
-[dependencies.pnet_macros_plugin]
-optional = true
-path = "pnet_macros_plugin"
-version = ">=0.1"
+clippy = { version = ">=0.0", optional = true }
+syntex = { version = "0.42", optional = true }
+pcap = { version = "0.6", optional = true }
+netmap_sys = { version = ">=0.0", optional = true, features = ["netmap_with_libs"] }
 
-[dependencies.pnet_macros_support]
-path = "pnet_macros_support"
-version = ">=0.1"
 
 [dev-dependencies]
 time = ">=0.1"
 
 [build-dependencies]
-glob = "0.2.*"
-syntex = { version = "0.42.*", optional = true }
```

Comment:
`0.x.*` is equivalent to `0.x`. I guess it's subjective which one is preferred. But I have not seen the asterisk version used almost anywhere. Not a big deal for me, it just happened to be removed when I was at it.

Reply:
I prefer the `.*` syntax, because it's far clearer (to me at least) which versions are supported.

When I read `version = "0.42"` it's not clear to me whether that means "exactly 0.42.0" or "0.42.*" or even ">=0.42 && <= 1.0".

Given that most crates don't use the `*` I'm not too bothered if you want to leave it out.

Reply:
I see your point, it's a very valid one. But if you are not too bothered I'm going to leave it, so I don't have to change again. And so it looks more in line with what is common in the ecosystem. I have to go look at the definition of version specifications every time anyone uses the `^`, `~` or `*` because those are the ones I'm not used to :D

Reply:
I also hate all those sigils - I wish they'd removed them when they removed all the other sigils in Rust.

Leaving it is fine.